src/libraries/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImplHelpers.cs

```diff
@@ -114,6 +114,8 @@ internal int LinePos
                     return charPos - lineStartPos;
                 }
             }
+
+            internal bool StartsWith(string prefix) => chars.AsSpan(charPos).StartsWith(prefix);
```

Comment:
@eiriktsarpalis and I may disagree on this point, but I think it'd be better without this helper; make it clear at the call site exactly what's being compared.  Plus, this could potentially have a perf impact, as it's causing the "local" chars and "charPos" to be lifted to a closure (though a value type) in order to be passed into the local function.

Reply:
Okay - I will remove it.